<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Render</class>
 <widget class="QWidget" name="Render">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>渲染</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <layout class="QVBoxLayout" name="canvas_layout"/>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="frame_label">
     <property name="text">
      <string>Frame</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSlider" name="frame_slider">
     <property name="value">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="start_button">
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="stop_button">
     <property name="text">
      <string>停止</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
